packages/replay-worker/vendor/index.js

```diff
@@ -0,0 +1,6 @@
+import workerString from './worker';
```

Comment:
Not sure if `vendor` is the best name, but these are basically pre-built files.
We need this because `workerString` is built by rollup at build-time, so we can't reference it at dev-time. I figured this is simple enough code to just pipe this through without any build step for now.

Note that this is transparent for the replay package, it doesn't really care about how we build stuff here.

Reply:
`static`? not sure if that's any better

Reply:
I'll just merge it as `vendor` for now and we can revisit this later, this is purely internal anyhow.